companies/digitalo.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "digitalo",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Re-In Retail International GmbH",
+    "address": "Re-In Retail International GmbH\nNordring 98a\n90409 Nürnberg\nDeutschland",
+    "phone": "+49 911 27430933",
+    "fax": "+49 911 27430934",
+    "email": "datenschutz@digitalo.de",
```

Comment:
As this email is specific for privacy-related requests, we can set the `suggested-transport-medium` to `email`.